static_ips_config.json

```diff
@@ -0,0 +1,44 @@
+[
+  {
+    "mac": "52:54:00:c2:f4:4d",
```

Comment:
can we generate it in the code? just get list of ips generate some random static mac and to +1 to every vm?

Reply:
fixed